HW04/anna_tsekhanskaya/lib/student.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'time'
+
+class Student
```

Comment:
Please, look at the comments to the Mentor class and try to correct this one accordingly